admin.adoc

```diff
@@ -41,6 +41,47 @@ Generating Django Secret key (used for xsrf protection etc)
 db@host:$
 --------------------------------------------------------------------
 
+To Encrypt and password the code & driver signing certificates
+
+1. Copy the keys (PrivateKeys.executable_signing_private_key & PrivateKeys.driver_signing_private_key) from the current GRR configuration file (whichever you’re using)
+ 
+ - /etc/grr/grr-server.yaml
+ - /etc/grr/server.local.yaml
+
+2. Save these two keys as new text files temporarily. You’ll need to convert the key text to the normal format by removing the leading whitespace and blank lines:
+
+ cat <original.exe.private.key.txt> | sed 's/^  //g' | sed '/^$/d’ > <clean.exe.private.key>
```

Comment:
Maybe change to
sed 's/^  //g' <original.exe.private.key.txt> | sed '/^$/d' > <clean.exe.private.key>

Note that all your last quotes are of the funny kind, maybe change them to normal single quotes in case people copy this (like me :) ).